cypress/e2e/Admin/create/createView.cy.js

```diff
@@ -107,5 +147,25 @@ describe('Create View', () => {
       cy.get(dataCyWrapper(CREATE_VIEW_DELETE_BUTTON_CY)).should('be.disabled');
       cy.get(dataCyWrapper(ADD_NEW_QUESTION_TITLE_CY)).should('be.visible');
     });
+
+    it('Add question', () => {
```

Comment:
What's the difference with the one above? Just so we can have a more detailed title.